agent/consul/coordinate_endpoint_test.go

```diff
@@ -42,6 +42,7 @@ func verifyCoordinatesEqual(t *testing.T, a, b *coordinate.Coordinate) {
 }
 
 func TestCoordinate_Update(t *testing.T) {
+	t.Parallel()
 	name := fmt.Sprintf("Node %d", getPort())
```

Comment:
Working on it one at a time. It is gone now since I've augmented the previous change to create unique node names also to other places. There are still a couple of hold-outs though.